zadanie_domowe_MB.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+funkcja_kalkulator () {
+	echo "Podaj pierwsza wartosc liczbowa"
+	read a
+	echo "podaj druga wartosc liczbowa"
+	read b
+	echo "Dodawanie: $((a + b))"
+	echo "Mnożenie: $((a * b))"
+}
+
+funkcja_petla () {
+	echo "Podaj liczbę mniejszą niż 10"
+	read a
+	if (( $a < 10 )); then
```

Comment:
w przypadku operacji arytmetycznych `(( ... ))` nie musisz używać '$'
```suggestion
	if (( a < 10 )); then
```